tekton/publish-customtask.yaml

```diff
@@ -140,9 +140,8 @@ spec:
         IMAGES_PATH=${IMAGES_PATH}/$(params.subfolder)
       fi
 
-      for d in cmd/*
-      do
-        IMAGES="${IMAGES} ${IMAGES_PATH}/${d}:$(params.versionTag)"
+      for d in controller; do
+        IMAGES="${IMAGES} ${IMAGES_PATH}/cmd/${d}:$(params.versionTag)"
       done
```

Comment:
Some custom tasks define their own custom resource and thus have a webhook, so I don't think this will work in general.